staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -769,3 +769,13 @@ func ExpectNoDiff(t *testing.T, msg string, expected, got interface{}) {
 		}
 	}
 }
+
+const dummyPrefix = "adapter"
+
+func EncodeContinueOrDie(key string, resourceVersion int64) string {
```

Comment:
No callers want to pass an invalid prefix and fail to encode, so we inject a dummy one to hide that detail from them.